packages/web-react/config/rollup.config.js

```diff
@@ -1,6 +1,7 @@
+import path from 'path';
 import resolve from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
```

Comment:
Why? Please, add at least some explaining comment.

Reply:
Actually I am not sure what part of the code you want an explanation for. The comment is added because require is also sort of import and the lint also contain rule for no empty line before import .

Reply:
I mean this one.

```
// eslint-disable-next-line import/order
import { terser as minify } from 'rollup-plugin-terser';
```

I do not get why it is disabled for ESLint check of the `import/order`.

Reply:
Because if you remove ` // eslint-disable-next-line import/order` there is an es-lint rule:

`ESLint: There should be no empty line between import groups (import/order)`

Itś because the following `const entryPoints = require('../scripts/entryPoints')`;  has empty line before